src/components/Quiz.js

```diff
@@ -18,19 +18,56 @@ const Quiz = () => {
     setQuestions(quizData);
   }, [quizName]);
 
-  const handleAnswerSelect = e => {
-    const selectedAnswer = { [e.target.name]: Number(e.target.value) };
+  const handleCheckboxAnswers = (e, answers) => {
+    const questionIds = Object.keys(answers || {});
+    const hasQPreviouslySelectedAnswers =
+      questionIds && questionIds.includes(e.target.name);
+
+    let checkedAnswer;
+    if (hasQPreviouslySelectedAnswers) {
+      const previousSelectedAnswers = answers[e.target.name];
+      const isAnswerPreviouslySelected = previousSelectedAnswers.includes(
+        Number(e.target.value),
```

Comment:
We have lots of `Number(e.target.value)` in this function. Let's extract it into a variable

Reply:
Done! but I'm not sure that variable name is a good name or not

Reply:
The variable name is great 👍